PeerioTechnologies/peerio-mobile
# fix: material icon color [branch ch5729]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/5729/mobile-update-material-icon-code-colour
#### Testing instructions  
Verify that colors of custom icons and material icons are the same now (see screenshot in CH)

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
